backend/jobsearch/src/main/kotlin/com/jobsearch/exception/Exceptions.kt

```diff
@@ -0,0 +1,6 @@
+package com.jobsearch.exception
+
+
+class NotFoundException(message: String): RuntimeException(message)
+class UserAlreadyExistsException(message: String): RuntimeException(message)
+class ForbiddenException(message: String): RuntimeException(message)
```

Comment:
Please use https://docs.oracle.com/javaee%2F7%2Fapi%2F%2F/javax/ws/rs/NotFoundException.html and https://docs.oracle.com/javaee%2F7%2Fapi%2F%2F/javax/ws/rs/ForbiddenException.html